db/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/db/InntektsmeldingRepository.kt

```diff
@@ -36,82 +38,131 @@ class InntektsmeldingRepository(
         forespoerselId: UUID,
         inntektsmeldingDokument: Inntektsmelding,
     ) {
-        val requestTimer = requestLatency.labels("lagreInntektsmelding").startTimer()
```

Comment:
Diffen i denne fila var ikke lett å lese, nå som jeg har lagt til et par funksjoner samtidig som jeg har tatt i bruk `recordTime` fra `felles/Metrics` _og_ renamet noen av dem 🙈 Skal prøve å flytte litt på de nye metodene for å se om det blir litt enklere å se hva som er gjort.